app/redux/ducks/classify.js

```diff
@@ -173,11 +187,19 @@ export default function reducer(state = initialState, action = {}) {
       const { goldStandardMode } = state;
       const { project } = action.payload;
       const { workflow } = state;
+      let { lastInterventionUUID} = state;
```

Comment:
I didn't take into account if the user is actually opted into the interventions and thus saw the message. I think this variable needs to be reflect this state before propagating to the `createNewClassification` method. Thoughts?

Reply:
If I'm understanding this question correctly, there are two pieces of information you need to know? Am I also correct in understanding this to mean that there can be a last intervention UUID, but the user may not have actually seen the intervention? If that's the case, then perhaps the state can be an object and the UUID and seen state is explicitly stored. Something like?: `{ lastIntervention: { UUID: '', seen: boolean } }` and this also gets recorded in the classification metadata?

Reply:
If you call actions from the `Intervention` component, I think that guarantees they will only be called for volunteers who have opted to see interventions.

Reply:
Reading through this some more, `lastInterventionUUID` is set when the `Intervention` component unmounts, so I think it can only be truthy here if an intervention was shown to the volunteer.

If I'm understanding these changes correctly:

1. I see an intervention and continue.
2. The intervention UUID is stored and the seen intervention is cleared.
3. When the next subject loads, `createNewClassification` is called, the UUID is added to the new classification and the stored UUID is cleared.